association.go

```diff
@@ -1855,7 +1858,7 @@ func (a *Association) handleSack(d *chunkSelectiveAck) error {
 		a.setRWND(d.advertisedReceiverWindowCredit - bytesOutstanding)
 	}
 
-	err = a.processFastRetransmission(d.cumulativeTSNAck, htna, cumTSNAckPointAdvanced)
+	err = a.processFastRetransmission(d.cumulativeTSNAck, d.gapAckBlocks, htna, cumTSNAckPointAdvanced)
```

Comment:
Don't know the protocol, but just reading this, can this just pass in the `end` of the last entry? Any reason for passing in the slice?

Reply:
Guess, you will have to do the check for length here rather than inside the function. Maybe, six of one and half-a-dozen of other case?

Reply:
No difference but I prefer to put the code and comment together to understand `all TSNs reported missing from the SACK (gapAckBlocks)`

Reply:
Thank you @cnderrauber